gui/default/syncthing/core/syncthingController.js

```diff
@@ -97,6 +97,7 @@ angular.module('syncthing.core')
                 }
 
                 $scope.version = data;
+                $scope.version.isDevelopmentVersion = data.version.indexOf('dirty')>0;
```

Comment:
This is too strict. I think a check for the presence of a "-" in the version would be enough.